crates/staking/README.md

```diff
@@ -0,0 +1,131 @@
+# Staking Pallet
+
+This pallet is used to manage joint funds for Vaults and Nominators.
+
+## Slashing
+
+We extend the [scalable reward distribution](https://solmaz.io/2019/02/24/scalable-reward-changing/) to account for the imbalance in Vault collateral. Instead of tallying the `reward_per_token` we increase `slash_per_token` to lazily calculate the proportion of collateral to subtract from a Vault or Nominator's stake on deposit or withdrawal.
```

Comment:
Could you be extra clear that slashing reduces the stake of _everybody_ in the pool proportionally?